bull-transformer-generator/transformer-generator-core/src/test/java/com/hotels/beans/generator/core/TransformerSpecTest.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beans.generator.core;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import java.lang.reflect.Type;
+
+import org.testng.annotations.Test;
+
+import com.hotels.beans.generator.core.sample.javabean.Destination;
+import com.hotels.beans.generator.core.sample.javabean.Source;
+import com.squareup.javapoet.MethodSpec;
+import com.squareup.javapoet.ParameterizedTypeName;
+import com.squareup.javapoet.TypeName;
+import com.squareup.javapoet.TypeSpec;
+
+/**
+ * Test for {@link TransformerSpec}.
+ */
+public class TransformerSpecTest {
+
+    private final TypeName iTransformer = TypeName.get(Transformer.class);
+
+    /**
+     * Should be named after source and destination types.
+     */
+    @Test
+    public void shouldBeNamedAfterSourceAndDestinationTypes() {
+        // given
```

Comment:
To keep the code consistency, please put the `// given`, `// when` and `// then` conditions in upper case, and leave an empty tow just after them.